bin/run-fisher.sh

```diff
@@ -5,6 +5,8 @@ if [ ! -f DeepSpeech.py ]; then
     exit 1
 fi;
 
+XDG_DATA_HOME=${XDG_DATA_HOME:="$HOME/DeepSpeech"}
```

Comment:
@kdavis-mozilla 

I know my linux install is not correct (not a full blown user but learning) and when i tested using

`XDG_DATA_HOME=${XDG_DATA_HOME:="$HOME/.local/share"}`

it physically created the path and saved the data in 

`~/.local/share/run-librivox$`

maybe that is what is supposed to happen but I thought it was supposed to save in the user's Deepspeech folder.

I will make the change as @reuben originally suggested it just did not test out the way I thought it was supposed to. I will fix this and re push.

At least the user will not have to face the TF shape error when running different scripts.

thanks.